spec/support/automation_example_group.rb

```diff
@@ -12,6 +12,7 @@ class << self
       RSpec.configure do |config|
         config.before(:suite) do
           puts "** Resetting ManageIQ domain"
+          Tenant.seed
```

Comment:
@kbrock 
We are importing a database during the reset, so we would need the default tenant to create the Domains.